src/proto/generation.proto

```diff
@@ -22,6 +23,7 @@ enum ArtifactType {
   ARTIFACT_MASK = 7;
   ARTIFACT_LATENT = 8;
   ARTIFACT_TENSOR = 9;
+  ARTIFACT_DEPTH = 10;
```

Comment:
Can you tell me what `ARTIFACT_DEPTH` is? Is it a `Tensor`?

Reply:
PNG data. ARTIFACT_DEPTH+tensor format would be nice for chain from transform server depth_calc to transform server 3d resample but can accomplish that with requesting ARTIFACT_TENSOR from depth_calc.

Reply:
To use the depth conditioned Stable Diffusion model (engine id `stable-diffusion-depth-v2-0` in animation environment) an init image (ARTIFACT_IMAGE) and init depth ARTIFACT_DEPTH) are passed in prompt.